docs/CONTRIBUTING.md

```diff
@@ -103,7 +103,7 @@ this:
   [`pr-deploy.yaml`](https://github.com/coder/coder/actions/workflows/pr-deploy.yaml)
   GitHub Action workflow:
 
-  <Image src="./images/deploy-pr-manually.png" alt="Deploy PR manually" width="350px" align="center" />
+  <Image src="./images/deploy-pr-manually.png" alt="Deploy PR manually" height="348px" align="center" />
```

Comment:
348. Nice.

Reply:
apparently `height` is the answer that doesn't get overridden 🤷 

![2024-11-08_12-56-30](https://github.com/user-attachments/assets/e4f5b451-c77b-4a6a-bfe2-f0b7c69b62b0)

Reply:
shoutout to that HTML 4 book I had back in 2000

Reply:
loooks way better, Thank you